test/strings/search.jl

```diff
@@ -408,3 +408,13 @@ for T = (UInt, BigInt)
         @test findprev(isletter, astr, T(x)) isa Int
     end
 end
+
+# issue 37280
+let A = [0x40, 0x52, 0x62, 0x52, 0x62]
+    @test findfirst([0x99], A) === nothing
+    @test findfirst([0x52], A) == 2:2
+    @test findfirst([0x52, 0x62], A) == 2:3
+    @test findnext([0x52, 0x62], A, 2) == 2:3
+    @test findnext([0x52, 0x62], A, 3) == 4:5
```

Comment:
This is better for ranges as it allows testing the bounds of empty ranges too:
```suggestion
    @test findfirst([0x52], A) === 2:2
    @test findfirst([0x52, 0x62], A) === 2:3
    @test findnext([0x52, 0x62], A, 2) === 2:3
    @test findnext([0x52, 0x62], A, 3) === 4:5
```